__tests__/app.test.js

```diff
@@ -394,4 +394,29 @@ describe("DELETE: /api/comments/:comment_id", () => {
     expect(msg).toBe("Bad request")
     })
   })
+})
+describe("GET /api/users", () => {
+  test("200: responds with an array of user objects", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body: {users}})=> {
+      users.forEach((user)=> {
+        expect(user).toMatchObject({
+          username: expect.any(String),
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  })
+  test("404: responds with error when typo made in url slug /users", () => {
```

Comment:
This test should not be in this describe block. You only need one test to check for an endpoint not existing and it does not relate to any specific endpoint so should be in its own describe block.